weave-js/src/components/FancyPage/useProjectSidebar.ts

```diff
@@ -161,7 +161,11 @@ export const useProjectSidebar = (
           {
             type: 'button' as const,
             name: 'Evals',
-            slug: 'weave/evaluations',
+            slug: 'weave/evaluations/evals',
+            additionalSlugs: [
+              'weave/evaluations/monitors',
```

Comment:
In general i think it is good to have the URL scheme match the application architecture. However, in this case, I am not sure that `monitors` will like under `evaluations` for the foreseeable future. I might opt to have `monitors` be a top-level path allowing us to iterate a bit more on application architecture before committing to this nesting.